python_src/src/lamp_py/performance_manager/l1_cte_statements.py

```diff
@@ -89,10 +86,11 @@ def get_static_trips_cte(
             ),
         )
         .join(
-            StaticCalendar,
+            ServiceIdDates,
             sa.and_(
-                StaticStopTimes.timestamp == StaticCalendar.timestamp,
-                StaticTrips.service_id == StaticCalendar.service_id,
+                StaticStopTimes.timestamp == ServiceIdDates.timestamp,
+                StaticTrips.service_id == ServiceIdDates.service_id,
+                StaticTrips.route_id == ServiceIdDates.route_id,
```

Comment:
This is our updated `service_id` matching to make sure we are pulling the correct services for a given service date.